sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2471,6 +2471,193 @@ async Task<Page<DocumentHealthcareResult>> NextPageFunc(string nextLink, int? pa
+        /// completion will contain layout elements extracted from the form.</returns>
+        public virtual AnalyzeOperation StartAnalyzeOperationBatch(IEnumerable<TextDocumentInput> documents, AnalyzeOperationOptions options, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(documents, nameof(documents));
+            options ??= new AnalyzeOperationOptions();
+            MultiLanguageBatchInput documentInputs = ConvertToMultiLanguageInputs(documents);
+
+            return StartAnalyzeOperationBatch(documentInputs, options, cancellationToken);
+        }
+
+        /// <summary>
+        /// Recognizes Analyze Operation async.
+        /// </summary>
+        /// <param name="documents"></param>
+        /// <param name="options"></param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A <see cref="AnalyzeOperation"/> to wait on this long-running operation.  Its <see cref="AnalyzeOperation.Value"/> upon successful
+        /// completion will contain layout elements extracted from the form.</returns>
+        public virtual async Task<AnalyzeOperation> StartAnalyzeOperationBatchAsync(IEnumerable<TextDocumentInput> documents, AnalyzeOperationOptions options, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(documents, nameof(documents));
+            options ??= new AnalyzeOperationOptions();
+            MultiLanguageBatchInput documentInputs = ConvertToMultiLanguageInputs(documents);
+
+            return await StartAnalyzeOperationBatchAsync(documentInputs, options, cancellationToken).ConfigureAwait(false);
+        }
+
+        private AnalyzeOperation StartAnalyzeOperationBatch(MultiLanguageBatchInput batchInput, AnalyzeOperationOptions options, CancellationToken cancellationToken = default)
+        {
+            options ??= new AnalyzeOperationOptions();
+
+
+            JobManifestTasks tasks = new JobManifestTasks();
+
+            AnalyzeBatchInput analyzeDocumentInputs = new AnalyzeBatchInput(batchInput, tasks);
+
+            if (options.PiiTaskParameters != null)
+            {
+                tasks.EntityRecognitionPiiTasks = new List<PiiTask>() { new PiiTask() { Parameters = options.PiiTaskParameters } };
+            }
+            if (options.EntitiesTaskParameters != null)
+            {
+                tasks.EntityRecognitionTasks = new List<EntitiesTask>() { new EntitiesTask() { Parameters = options.EntitiesTaskParameters } };
+            }
+            if (options.KeyPhrasesTaskParameters != null)
+            {
+                tasks.KeyPhraseExtractionTasks = new List<KeyPhrasesTask>() { new KeyPhrasesTask() { Parameters = options.KeyPhrasesTaskParameters } };
+            }
+
+            if (options.DisplayName != null)
+            {
+                analyzeDocumentInputs.DisplayName = options.DisplayName;
```

Comment:
curious, what will happen if there is no validation and the SDK sends `analyzeDocumentInputs.DisplayName = null` ?

Reply:
So currently in the swagger the name is required property and its not returned from the service if not sent. I will make changes to remove the validations on Name.

Reply:
thank you for explaining that :)

Reply:
hmm I see so looks like the swagger needs to be updated because if this is optional as input then it is optional as output.
then yeah, please look at my other comments about doing a temporary transformation. Once the service fixes this, we only need to remove the transformation